### PR TITLE
fix for TOF async calib

### PR DESCRIPTION
**Pull request by noferini**

### CODE REVIEW

**Comment by noferini** [comment]
@chiarazampolli 
I am currently running a test on the pilot beam 2021 calibration outputs of apass4 to validate all the steps (test ccdb).
If fine I will open a JIRA asking you to re-run it with production ccdb.

Note that for the next runs (apass1) I added a flag in tof-reco-workflow to specify that calibrations are not yet done (to be done in apass1). This will switch off the filter on problematic channels (loaded from ccdb of previous runs) since problematic channels should be defined after calibration.

**Comment by noferini** [comment]
PR is ready for the review.
Tests with 2021 pilot beam look fine.

**Comment by noferini** [comment]
failing checks not related to this PR

File: Detectors/TOF/base/src/CalibTOFapi.cxx [hunk 1]
```diff
@@ -272,6 +272,28 @@ float CalibTOFapi::getTimeCalibration(int ich, float tot)
 
 //______________________________________________________________________
 
+float CalibTOFapi::getTimeCalibration(int ich, float tot, float phase)
```
Comment (shahor02) [inline Detectors/TOF/base/src/CalibTOFapi.cxx:275]: this should be const right? Can be changed in next PR

**Review by shahor02 — APPROVED** [review]

File: Detectors/TOF/base/src/CalibTOFapi.cxx (hunk 1 above)
Comment (noferini) [reply Detectors/TOF/base/src/CalibTOFapi.cxx:275]: Yes, I forgot sorry

**Pull request merged**